warehouse/core/src/main/java/datawave/iterators/IterationInterruptedException.java

```diff
@@ -0,0 +1,15 @@
+package datawave.iterators;
+
+/**
+ * Exception thrown if an interrupt flag is detected.
+ */
+public class IterationInterruptedException extends RuntimeException {
```

Comment:
I might suggest customizing the name slightly to avoid IDE's pulling in accumulo's import of the class of the same name to avoid confusion later.